solutions/ids/endpoint/mirroring.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# Packet mirroring policy
+# 1 per vpc and per region minimum
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputePacketMirroring
+metadata:
+  name: host-project-id-endpoint-name-mirror # kpt-set: ${host-project-id}-${endpoint-name}-mirror
```

Comment:
see other comment regarding `host-project-id` in parent setters

Reply:
added "--" as recommended